card/mod/item/set/all/bar.rb

```diff
@@ -1,22 +1,25 @@
 include_set Abstract::BsBadge
 
 format :html do
-  view :mini_bar do
-    render_bar hide: [:bar_middle]
+  setting :bar_cols
+  setting :infobar_cols
+
+  view :infobar do
```

Comment:
I'm probably going to trip up on the fact that expanded_bar has an underscore and infobar doesn't.  How do you feel about `info_bar`?

Reply:
I'm not attached to `infobar`. `bar_with_middle` would be more 
self-explanatory.